analyses/tumor-normal-differential-expression/workflows/run_deseq_analysis_wf.cwl

```diff
@@ -75,26 +91,28 @@ steps:
       ram: ram
       cpus: cpus
     out: [results_dir]
-
   combine_output_files:
     run: ../tools/combine_output_files.cwl
     in:
       results_dirs: run_deseq2/results_dir
       output_basename: output_basename
-    out:
-      [combined_tsv, combined_jsonl]
-
+    out: [combined_tsv, combined_jsonl]
   convert_tsv_to_rds:
     run: ../tools/convert_tsv_to_rds.cwl
     in:
       combined_tsv: combine_output_files/combined_tsv
       output_basename: output_basename
     out: [merged_rds]
-
 $namespaces:
   sbg: https://sevenbridges.com
 hints:
-  - class: 'sbg:maxNumberOfParallelInstances'
-    value: 80
-  - class: 'sbg:AWSInstanceType'
-    value: r5.24xlarge
+- class: 'sbg:maxNumberOfParallelInstances'
+  value: 80
```

Comment:
Remind me, why is this number so high? Was it in case you had lots of disease comaprisons

Reply:
@migbro Yes, I have uploaded and run this version on Cavatica. 
You are also right about the high number of AWS instances used. With that many instances the overall execution time is less than an hour to run comparisons on the full set of combinations of normal and tumor histologies.